demo/site/src/util/graphQLClient.ts

```diff
@@ -29,7 +29,7 @@ export function createGraphQLFetch() {
             },
             headers: {
                 "x-relative-dam-urls": "1",
-                authorization: `Basic ${Buffer.from(`vivid:${process.env.API_PASSWORD}`).toString("base64")}`,
+                authorization: `Basic ${Buffer.from(`vivid:${process.env.API_BASIC_AUTH_SYSTEM_USER_PASSWORD}`).toString("base64")}`,
```

Comment:
@johnnyomair We could also throw a hard error here, what do you think?

Reply:
Yes, we should throw an error here if the environment variable isn't set.

Reply:
f2f5033fa0dfe04d0880ec1a30809cc0ed213476